201-blob-storage-private-endpoint/scripts/nslookup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Variables
+storageAccountName=$1
+
+# Parameter validation
+if [[ -z storageAccountName ]]; then
+    echo "storageAccountName parameter cannot be null or empty"
+    exit 1
+fi
+
+# Update the system
+sudo apt-get update -y
+
+# Upgrade packages
+sudo apt-get upgrade -y
+
+# Run nslookup to verify that the <storage-account>.blob.core.windows.net public hostname of the storage account 
+# is properly mapped to <storage-account>.privatelink.blob.core.windows.net by the private DNS zone
+# and the latter mapped to the private address by the A record
+nslookup "$storageAccountName.blob.core.windows.net"
```

Comment:
you need to pass in the suffix for the storage account... or just pass in the whole FQDN

Reply:
Fixed per your request, no hardcoded values!